### PR TITLE
Add flask-apscheduler package

### PR DESCRIPTION
**Pull request by edetec**

### CODE REVIEW

File: recipes/flask-apscheduler/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ pkg_name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```
[the remark this answers is not in the document]
Comment (edetec) [reply recipes/flask-apscheduler/meta.yaml]: I think so

File: recipes/flask-apscheduler/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    to platform specific schedulers, such as the cron daemon or the Windows task scheduler. Please 
+    note, however, that APScheduler is not a daemon or service itself, nor does it come with any 
+    command line tools. It is primarily meant to be run inside existing applications. That said, 
+    APScheduler does provide some building blocks for you to build a scheduler service or to run a 
+    dedicated scheduler process.
+  doc_url: https://apscheduler.readthedocs.io
+  dev_url: https://github.com/viniciuschiele/flask-apscheduler
+
+extra:
+  recipe-maintainers:
```
[the remark this answers is not in the document]
Comment (edetec) [reply recipes/flask-apscheduler/meta.yaml]: @igortg and @lvoliveira, do you agree to be co-maintainers?

[the remark this answers is not in the document]
Comment (lvoliveira) [reply recipes/flask-apscheduler/meta.yaml]: yes

**Comment by synapticarbors** [comment]
I'm going to make the assumption that given @igortg's work on a large number of other conda-forge packages, adding this one won't be an issue.

**Pull request merged**